js/functions.js

```diff
@@ -2211,3 +2213,109 @@ function getEventDescription(isAerobatics, locationName, minutes) {
     var desc = 'יחל ב';
     return `${desc}${locationName} בעוד ${minutes} דקות`;
 }
+
+
+
+(function () {
+    var userLoc = null;
+    navigator.geolocation.watchPosition(function(newLoc){
+        userLoc = newLoc;
+        userLoc = {lon: userLoc.coords.longitude, lat: userLoc.coords.latitude};
+    });
+
+    var aircraftsName = [       
+        {'hebName':'ברק','engName':'barak'},
+        {'hebName':'אדיר','engName':'adir'},
+        {'hebName':'עטלף','engName':'atalef'},
+        {'hebName':'בז','engName':'baz'},
+        {'hebName':'בוינג747','engName':'boeing747'},
+        {'hebName':'','engName':'c17'},
+        {'hebName':'','engName':'airtractor'},
+        {'hebName':'','engName':'c295'},
+        {'hebName':'עפרוני','engName':'efroni'},
+        {'hebName':'עיטם','engName':'eitam' },
+        {'hebName':'F-16','engName':'f16'},
+        {'hebName':'קרנף','engName':'karnef'},
+        {'hebName':'לביא','engName':'lavi'},
+        {'hebName':'נחשון','engName':'nahshon'},
+        {'hebName':'פתן','engName':'peten'},
+        {'hebName':'רעם','engName':'raam'},
+        {'hebName':'ראם','engName':'reem'},
+        {'hebName':'סייפן','engName':'saifan'},
+        {'hebName':'שרף','engName':'saraf'},
+        {'hebName':'שחף','engName':'shahaf'},
+        {'hebName':'שמשון','engName':'shimshon'},
+        {'hebName':'שמשון בריטי','engName':'shimshon-britian'},
+        {'hebName':'שמשון אוסטרי','engName':'shimshon-ostrian'},
+        {'hebName':'סופה','engName':'sufa'},
+        {'hebName':'צופית','engName':'tzufit'},
+        {'hebName':'ינשוף','engName':'yanshuf'},
+        {'hebName':'יסעור','engName':'yasur'}
+    ];
+
+    function notifyUserIfNear(currentLocation, aircraft) {
+        if (userLoc) {
+            currentLocation = {lon: currentLocation.lng, lat: currentLocation.lat};
+            if(haversineDistance(userLoc,currentLocation) < 1000)
+            {
+                var name = aircraft.name;
+                document.getElementById("gottoVoiceMessagePopup").style.display = "block";
```

Comment:
A general comment for all of document.getElementById - it is preferred to use jQuery syntax instead to find object hide or show them.

Reply:
We will change all the getElementById occurrences. @mayPeleg

Reply:
Note: getElementById is almost 3 times faster  than jQuery.
https://www.measurethat.net/Benchmarks/Show/3/0/jquery-by-id-vs-documentgetelementbyid

Reply:
Done